pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -602,8 +602,7 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(ctx context.Context,
 	return statuses, nil
 }
 
-func toKubeContainerStatus(status *runtimeapi.ContainerStatus, runtimeName string) *kubecontainer.Status {
-	annotatedInfo := getContainerInfoFromAnnotations(status.Annotations)
+func toKubeContainerStatus(status *runtimeapi.ContainerStatus, runtimeName string, annotatedInfo *annotatedContainerInfo) *kubecontainer.Status {
```

Comment:
I'm curious about one thing. `annotatedInfo`  Where is this parameter used?

Reply:
https://github.com/kubernetes/kubernetes/blob/850bfd9aa92423696dbb6de2070ab7494fdd99b1/pkg/kubelet/kuberuntime/kuberuntime_container.go#L646-L647